content/sign.json

```diff
@@ -5,6 +5,6 @@
   "titleFail": "Signing failed, see the errors below",
   "form": {
     "title": "Sign your Self Description",
-    "subtitle": "Input your self description and we will sign it with a deltaDAO test-key. Need inspiration for your self-descriptions? Check out our [participant self-description](https://raw.githubusercontent.com/deltaDAO/files/main/sd_examples/participant_sd.json) and [service self-description](https://raw.githubusercontent.com/deltaDAO/files/main/sd_examples/service_offering_experimental_sd.json) examples."
+    "subtitle": "Input your self description and we will sign it with a deltaDAO test-key. Need inspiration for your self-descriptions? Check out our [participant self-description](https://raw.githubusercontent.com/deltaDAO/self-description-signer/main/config/self-description-examples/participant-self-description.json) and [service self-description](https://raw.githubusercontent.com/deltaDAO/self-description-signer/feat/main/config/self-description-examples/service-offering-self-description.json) examples."
```

Comment:
The new links to the SD examples refer to the main branch so they will be dead until https://github.com/deltaDAO/self-description-signer/tree/feat/adjust-to-2206 will be merged